package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tunnel-quest-bot",
   "version": "1.1.2",
-  "lockfileVersion": 2,
```

Comment:
I just regenerated the entire package-lock file for the first time in a while, and this is what it did -- I guess the new version doesn't require the dependencies section? Or was that something we added manually?